salt/proxy/junos.py

```diff
@@ -55,14 +58,17 @@
     )
     from ncclient.operations.errors import TimeoutExpiredError
     from ncclient.transport.third_party.junos.ioproc import IOProc
-except ImportError:
+
+## DGM except ImportError:
```

Comment:
This one feels like an unnecessary comment? The exception caught say's it all?

Reply:
fixed